hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/PingHandler.java

```diff
@@ -39,6 +39,6 @@
     @Produces({ APPLICATION_JSON })
     @ApiOperation("A dummy operation returning the current date on the server.")
     public Response ping() {
-        return Response.ok(new StringValue(new Date().toString())).build();
+        return Response.ok(new Date().toString()).build();
```

Comment:
Is this not changing the API?
I believe, before this spewed:
```json
{ "value": "...date..."}
```
While after this change the output would be:
```
...date...
```